test/jdk/java/awt/image/multiresolution/MultiDisplayTest.java

```diff
@@ -23,61 +23,104 @@
 
 
 /*
-  @test
-  @bug 8142861 8143062 8147016
-  @summary Check if multiresolution image behaves properly
-           on HiDPI + non-HiDPI display pair.
-  @author a.stepanov
-  @library /test/lib
-  @build jdk.test.lib.Platform
-  @run applet/manual=yesno MultiDisplayTest.html
-*/
-
-
-import java.applet.Applet;
-import java.awt.*;
-import java.awt.event.*;
-import java.awt.image.*;
+ * @test
+ * @bug 8142861 8143062 8147016
+ * @library /java/awt/regtesthelpers /test/lib
+ * @build PassFailJFrame jdk.test.lib.Platform
+ * @summary Check if multiresolution image behaves properly
+ *         on HiDPI + non-HiDPI display pair.
+ * @run main MultiDisplayTest
+ */
```

Comment:
We usually move this block just before a class declaration, for better readability.

Reply:
Moved.